File system scan not working in v1.13.0

#### Describe the bug

Since the release of v1.13.0 our file system scans have started failing as they seem be trying to do an image scan instead without any docker/image details.
The version of Trivy itself hasn't changed since it was working - v0.61.0.

Previously, the task was invoking ```/opt/hostedtoolcache/trivy/0.61.0/x64/trivy fs``` but now it is invoking ```/opt/hostedtoolcache/trivy/0.61.0/x64/trivy image```.

Seems this might be related to #140 somehow.

#### Agent details

* Type: Microsoft-Hosted Agent
* OS: Linux, (ubuntu-latest)
* Version: 3.251.0

#### Task Version

[v1.13.0](https://github.com/aquasecurity/trivy-azure-pipelines-task/releases/tag/v1.13.0)

#### Task Inputs

<!-- Provide your task inputs without any sensitive data -->
<!-- At least specify run method. eg docker/pre-installed binary/installed by task-->
```yaml
- task: trivy@1
        displayName: "Run Trivy secret scan"
        inputs:
          docker: false
          path: $(Build.SourcesDirectory)
          options: "--scanners secret"
          exitCode: "1"
```

#### Error message

```log
Requested Trivy version to install: latest
Resolved version: v0.61.0
Downloading: https://github.com/aquasecurity/trivy/releases/download/v0.61.0/trivy_0.61.0_Linux-64bit.tar.gz
Extracting archive
/usr/bin/tar xC /home/vsts/work/_temp/4e05009e-e001-42c9-95b5-0743215e4079 -f /home/vsts/work/_temp/338acddb-6e2e-4e5e-87d3-1fcf7c7b3ba2

Caching tool: trivy 0.61.0 x64
Prepending PATH environment variable with directory: /opt/hostedtoolcache/trivy/0.61.0/x64
Configuring options for image scan...
/opt/hostedtoolcache/trivy/0.61.0/x64/trivy image --exit-code 1 --format json --output /home/vsts/work/_temp/trivy/trivy-results-555e1d75-4310-48c0-abf3-fcec59b1c7f8.json --scanners secret /home/vsts/work/1/s
2025-04-05T18:45:40Z	INFO	[secret] Secret scanning is enabled
2025-04-05T18:45:40Z	INFO	[secret] If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2025-04-05T18:45:40Z	INFO	[secret] Please see also https://trivy.dev/v0.61/docs/scanner/secret#recommendation for faster secret detection
##[error]Failed: Trivy detected problems.
2025-04-05T18:45:41Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scan service: unable to initialize an image scan service: unable to find the specified image "/home/vsts/work/1/s" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (/home/vsts/work/1/s): Error response from daemon: No such image: home/vsts/work/1/s:latest
	* containerd error: parse error: invalid reference format
	* podman error: unable to inspect the image (/home/vsts/work/1/s): failed to find image home/vsts/work/1/s: home/vsts/work/1/s: No such image
	* remote error: GET https://index.docker.io/v2/home/vsts/work/1/s/manifests/latest: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:home/vsts/work/1/s Type:repository]]


##[error]Trivy seems to have failed so no output path to generate reports from.
Done!

```

#### Previously working example

```log
Requested Trivy version to install: latest
Resolved version: v0.61.0
Downloading: https://github.com/aquasecurity/trivy/releases/download/v0.61.0/trivy_0.61.0_Linux-64bit.tar.gz
Extracting archive
/usr/bin/tar xC /home/vsts/work/_temp/a97ef0dc-c963-4f22-b885-1e7029781d98 -f /home/vsts/work/_temp/b227726c-3188-49b8-8cdb-1bc55b54b711

Caching tool: trivy 0.61.0 x64
Prepending PATH environment variable with directory: /opt/hostedtoolcache/trivy/0.61.0/x64
Configuring options for image scan...
/opt/hostedtoolcache/trivy/0.61.0/x64/trivy fs --exit-code 1 --format json --output /home/vsts/work/_temp/trivy/trivy-results-5d9b28cf-8f38-4e05-80ab-c31888d28140.json --scanners secret /home/vsts/work/1/s
2025-04-05T11:01:28Z	INFO	[secret] Secret scanning is enabled
2025-04-05T11:01:28Z	INFO	[secret] If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2025-04-05T11:01:28Z	INFO	[secret] Please see also https://trivy.dev/v0.61/docs/scanner/secret#recommendation for faster secret detection
2025-04-05T11:01:28Z	INFO	[secret] Loading the config file for secret scanning...	config_path="trivy-secret.yaml"
Done!

```

--- a/trivy-task/trivyV1/index.ts
+++ b/trivy-task/trivyV1/index.ts
@@ -104,6 +104,10 @@ function configureScan(runner: ToolRunner, inputs: TaskInputs, output: string) {
     runner.line(inputs.options);
   }
 
+  if (!scanTarget) {
+    throw new Error('No scan target specified.');
+  }
+
   runner.arg(scanTarget);
 }
 

--- a/trivy-task/trivyV1/task.json
+++ b/trivy-task/trivyV1/task.json
@@ -127,11 +127,11 @@
       "name": "type",
       "type": "pickList",
       "label": "Scan Type",
-      "defaultValue": "image",
       "required": false,
       "options": {
         "image": "Image",
-        "filesystem": "Filesystem"
+        "filesystem": "Filesystem",
+        "repo": "Repository"
       },
       "helpMarkDown": "The type of scan to perform, can be a filesystem scan or an image scan.",
       "groupName": "scanning"
@@ -140,7 +140,6 @@
       "name": "path",
       "type": "string",
       "label": "Path to scan",
-      "defaultValue": "",
       "required": false,
       "helpMarkDown": "The specified path will be scanned using an 'fs' scan type. If this option is used, the 'image' option cannot be set.",
       "groupName": "scanning",
@@ -150,7 +149,6 @@
       "name": "image",
       "type": "string",
       "label": "Image to scan",
-      "defaultValue": "",
       "required": false,
       "helpMarkDown": "The specified image will be scanned using an 'image' scan type. If this option is used, the 'path' option cannot be set.",
       "groupName": "scanning",

--- a/trivy-task/trivyV1/utils.ts
+++ b/trivy-task/trivyV1/utils.ts
@@ -10,8 +10,8 @@ export type TaskInputs = {
   debug?: boolean;
   devMode?: boolean;
   type?: string;
-  image: string;
-  scanPath: string;
+  image?: string;
+  scanPath?: string;
   scanners: string;
   severities: string;
   ignoreUnfixed?: boolean;
@@ -41,8 +41,8 @@ export function getTaskInputs(): TaskInputs {
     debug: task.getBoolInput('debug', false),
     devMode: task.getBoolInput('devMode', false),
     type: task.getInput('type', false) ?? '',
-    image: task.getInput('image', false) ?? '',
-    scanPath: task.getInput('path', false) ?? '',
+    image: task.getInput('image', false),
+    scanPath: task.getInput('path', false),
     scanners: task.getInput('scanners', false) ?? '',
     severities: task.getInput('severities', false) ?? '',
     ignoreUnfixed: task.getBoolInput('ignoreUnfixed', false),

